util/find-doc-nits

```diff
@@ -708,7 +708,7 @@ sub collectnames {
 
     my @foreign_names =
         map { map { s/\s+//g; $_ } split(/,/, $_) }
-        $podinfo{contents} =~ /=for\s+openssl\s+foreign\s+manuals:\s*(.*)\n\n/;
+        $podinfo{contents} =~ /=for openssl foreign manual (.*)\n/;
```

Comment:
Why?

Reply:
simpler syntax, more consisten with the other meta-comments.

Reply:
(see 'for openssl ifdef' for example)

Reply:
... ok.

I would push for the other way around, for several reasons:

1. yours is unforgiving.  One space mistake somewhere (one too few or one too many) and it won't catch.  Silently.
2. the construct allows for several manuals, hence plural

Personally, I prefer the colon to be there, but oh well.

(why I haven't said anything before?  A lapse, I suppose...)

Reply:
The "for ifdef" allows several words as well.  And if the syntax is wrong, then the line won't be parsed and it will show up as doc-link errors.  Once I fix this PR, that will be analogous to a doc-nits error so I'm not worried about getting the syntax wrong.  A followon PR can always change 'for manuals' and 'for ifdef' to whatever you want, of course :)

Reply:
Not sure I understand what you're saying. Should we be more unforgiving to those who write the pod files (more unforgiving than the real pod parsers)?

Reply:
Right now the concern is theoretical, and will be caught at build time.